airflow/providers/mongo/hooks/mongo.py

```diff
@@ -75,6 +74,23 @@ def __init__(self, mongo_conn_id: str = default_conn_name, *args, **kwargs) -> N
         self.client: MongoClient | None = None
         self.uri = self._create_uri()
 
+        self.allow_insecure = self.extras.pop("allow_insecure", "false").lower() == "true"
+        self.ssl_enabled = self.extras.get("ssl", "false").lower() == "true"
```

Comment:
NIT: As @Taragolis suggested:
```suggestion
        self.ssl_enabled = self.extras.get("ssl", "false").lower() == "true" or self.extras.get("tls", "false").lower() == "true" 
```

Reply:
(and also add docstring).

Reply:
Sure, making the changes